Azure-Samples/msdocs-django-postgresql-sample-app
# Create 0002_alter_review_rating.py

## Purpose

This repository currently does not contain all the migrations that correspond to the full schema, so Django gives you a warning when you run migrate with the current migrations. Specifically, it's missing the rating field on restaurant reviews table.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

*  Get the code
* `pip install -r requirements.txt`
* `python manage.py migrate`
* `python manage.py makemigrations`: you should see no migrations to make!

## What to Check

When you look at the database tables, there should be a rating field in restaurant reviews.

## Other Information

Another approach is to nuke the existing 0001 migration and just replace with a single migration file that includes the rating field from the getgo, but then anyone with the existing repo would have an issue updating, I think. (Probably not a big deal, so either approach is fine).
